src/typings.ts

```diff
@@ -175,7 +365,9 @@ export interface Token {
   access_token: string;
   refresh_token: string;
   token_type: string;
+  /** created at in seconds */
   created_at: number;
+  /** expiry in seconds */
```

Comment:
Is that seconds after created_at? If so, maybe we can be explicit about it.

Reply:
I copied the description from [the RFC](https://datatracker.ietf.org/doc/html/rfc6749#section-5.1)

> The lifetime in seconds of the access token.  
> For example, the value "3600" denotes that the access token will expire in one hour from the time the response was generated.

and added a link to the spec on the type.